### ISSUE TITLE
Incorrect variable set in doc.

### ISSUE THREAD

**Issue by AkhterAli** [saltstack/salt#25582]
In the following link this 
```python
import salt.config
import salt.loader

__opts__ salt.config.minion_config('/etc/salt/minion')
__grains__ = salt.loader.grains(__opts__)
print __grains__['id']
```
should be
```python
import salt.config
import salt.loader

__opts__ =  salt.config.minion_config('/etc/salt/minion')
__grains__ = salt.loader.grains(__opts__)
print __grains__['id']
```

http://docs.saltstack.com/en/2014.7/ref/clients/#salt.loader.grains

**Comment by jfindlay** [comment]
@AkhterAli, thanks for the report.  You are welcome to submit a pull request to fix it.

**Comment by cachedout** [comment]
Closed via #25633
